src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1614,12 +1635,7 @@ private void InitializeNodeLibrary()
             // Without the index files on disk, the dirReader cant be initialized correctly. So does the searcher.
             if (!IsTestMode)
             {
-                LuceneSearchUtility.dirReader = LuceneSearchUtility.writer?.GetReader(applyAllDeletes: true);
-                LuceneSearchUtility.Searcher = new IndexSearcher(LuceneSearchUtility.dirReader);
-
                 LuceneSearchUtility.writer?.Commit();
-                LuceneSearchUtility.writer?.Dispose();
-                LuceneSearchUtility.writer = null;
             }
```

Comment:
same comment here

Reply:
Fixed in commit: https://github.com/DynamoDS/Dynamo/pull/14120/commits/85334fcdf0a658d74b5692fdad74005b7c6978a2